src/views/migrationpolicies/list/components/MigrationPoliciesCreateButton/MigrationPoliciesCreateButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ListPageCreateDropdown } from '@openshift-console/dynamic-plugin-sdk';
+
+import { migrationPoliciesPageBaseURL } from '../../utils/constants';
+
+type MigrationPoliciesCreateButtonProps = {
+  kind: string;
```

Comment:
can you remove this prop and import MigrationPolicyModelRef instead? it will make it a bit more clear what is this actually, and it will allow you to use this with no prop which will be easier if needed elsewhere :)

Reply:
I am not sure if this is a good idea, because if you check the related parent components, they also take `kind` as a prop. If it was so simple and safe to do it, I am sure the parent components wouldn't get this at all, but used what you suggest.

Reply:
I looked in the code, and the parent is getting it free from the console, as it's a menu item.
Getting the kind its not a network request or IO action that takes time. It's a simple function. From the name of the component MigrationPoliciesCreateButton, I can see it's not a generic component, and it's dedicated to migration policy, u can emit the prop and use it as Aviv suggested or leave it as it is. There isn't a big difference here in this case, it simply makes it a bit cleaner if u getting it as Aviv suggested because the component doesn't depend on anything, but again there isnt a much difference.

Reply:
So then if there isn't much difference, shouldn't we omit getting kind also in parent components?

Reply:
Actually, I would leave it as it is.. u did it ok; I just wanted to let you both know a bit more information pros and cons of the suggest change.